test/test-webdriver-plus.ts

```diff
@@ -63,6 +88,27 @@ describe('webdriver-plus', () => {
       assert.equal(await root.findContent('.cls1', /K$/).getText(), 'OK');
     });
 
+    it('should wait for matching content with driver.findContentWait()', async function() {
+      // Add component after 500ms.
+      setTimeout(() => driver.executeScript(addDom, 'waitForContent'), 500);
+      // Wait for component being added.
+      assert.equal(await driver.findContentWait(1, '#waitForContent', /Foo/).getText(), 'Foo');
+    });
+
+    it('should wait for child content match with element.findContentWait()', async function() {
+      // Add component after 500ms.
+      setTimeout(() => driver.executeScript(addDom, 'elemWaitForContent', 'id1'), 500);
+      const root = await driver.find('#id1');
+      // Wait for component being added.
+      assert.equal(await root.findContentWait(1, '#elemWaitForContent', /Foo/).getText(), 'Foo');
+    });
+
+    it('should find the closest matching ancestor with element.findClosest()', async function() {
+      const child = await driver.find(".cls2");
+      await assert.match(await child.findClosest('#id1').getText(), /Hello/);
+      await assert.isRejected(child.findClosest('#id2'), /None.*match/);
```

Comment:
Maybe also ass `assert.equal(child.findClosest('#id2').isPresent(), false)`, just to make sure the right error is thrown for isPresent to understand it?